js/main.js

```diff
@@ -4,55 +4,63 @@ const { author } = bookForm;
 const addBtn = document.getElementById('addBtn');
 const booksContainer = document.querySelector('.booksContainer');
 
-let books = [];
-books = JSON.parse(localStorage.getItem('books')) !== null ? (books = JSON.parse(localStorage.getItem('books'))) : [];
-
-let storageBooks;
-const getLocalData = () => {
-  storageBooks = JSON.parse(localStorage.getItem('books'));
-};
-
-const showBooks = () => {
-  booksContainer.innerHTML = '';
-  getLocalData();
-  storageBooks.forEach((book) => {
-    const bookTemplate = `
-            <p>${book.title}</p>
-            <p>${book.author}</p>
-            <button type="button" class="remove" id="${book.id}">Remove</button>
-            <hr>
+class Book {
+  static books = [];
+
+  localStorageBooks;
+
+  static addBook = (e) => {
+    e.preventDefault();
+    this.books = JSON.parse(localStorage.getItem('books')) !== null ? (this.books = JSON.parse(localStorage.getItem('books'))) : [];
+
+    this.book = {
+      id: 0,
+      title: '',
+      author: '',
+    };
+
+    if (title.value === '' || author.value === '') {
+      return false;
+    }
+
+    this.book.title = title.value;
+    this.book.author = author.value;
+    this.book.id = this.books.length + 1;
+    this.books.push(this.book);
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to use array methods that do not mutate the array. For example, in this case, you can use the `concat` method instead of `push`.

Reply:
Thanks for pointing that out, @TedLivist.
I will follow your suggestion and improve it.